### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Shaheryar0054**
- Added styling for a popup window
- Added javascript functionality
- Added a new javascript file for a popup window
- Added images for a popup window

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -101,14 +101,14 @@ <h2 class="heading-4">Muti-Post Stories</h2>
...
+                <button class="see-project" id="prjsbtn">See Project</button>
             </div>
         </div>
     </section>
 
     <section id="main-container">
         <div class="item-1">
-            <button id="see-project">See Project</button>
+            <button class="see-button" id="prjsbtn">See Project</button>
         </div>
```
Comment (christianbiring1) [inline index.html]: At this stage, Your project is supposed to be rendered dynamically.
- [ ]  Kindly make the projects section is created dynamically with the information from the projects array (_all the HTML of the section is created when the page loads_). `The already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript.`

File: style.css [hunk 2]
```diff
@@ -692,6 +692,158 @@ body {
...
+  margin: 50px 20px;
+  padding: 16px;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: max-content;
+}
+
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure your pop-up model fits the screen size on the Desktop version. It is not supposed to fill all the `height`. As you can see in your design. The user is not even able to see the training part of the model.
Your design :point_down: 

Screenshot from 2022-10-21 14-28-15

Figma template :point_down: 

image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Shaheryar0054 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Great UI :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Populating the data for the exact project where the button was clicked(`image(picture), description`)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: style.css [hunk 3]
```diff
@@ -692,6 +692,163 @@ body {
...
+  padding: 16px;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 631px;
+  margin-left: 171px;
+  margin-top: 35px;
+}
```
Comment (Abenezer-Tilahun) [inline style.css:718]: - [ Optional ] Good job for addressing the previous reviewer suggestion, But  I recommend  to adjust ``margin-left:100px;` to make the popup is responsive, it fits desktop sizes properly.  

Your Project Popup at `margin-left: 171px;` look likes this : 
Capture

I think to fit well at `margin-left: 100px;` Look likes this : 

Capture

File: style.css [hunk 4]
```diff
@@ -1015,4 +1172,41 @@ body {
...
+    margin: 50px 20px;
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 718px;
+    flex-direction: column;
+    width: auto;
+  }
```
Comment (Abenezer-Tilahun) [inline style.css:1188]: - [ optional ] Please make sure that the popup is responsive, it fits mobile  sizes properly. So I recommend you to adjust `margin: 50px 20px;`  to make it responsive.

The Popup for mobile size at `margin: 50px 20px;` look likes this : 

Capture

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Shaheryar0054 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**